README.md

```diff
@@ -53,4 +53,44 @@ Finally, run the deployment engine locally:
 ## SonarQube
 
 Sonar analysis is performed by Github Actions on the code repository for this
-project.  Results available at [sonarcloud.io](https://sonarcloud.io/project/overview?id=aoc-aap-test-installer)
\ No newline at end of file
+project.  Results available at [sonarcloud.io](https://sonarcloud.io/project/overview?id=aoc-aap-test-installer)
+
+## Developer Setup
+
+### Prequisties
+
+- Linux or MacOS
+- [VS Code](https://code.visualstudio.com/download)
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+- [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [NVM](https://github.com/nvm-sh/nvm)
+- [Python](https://www.python.org/downloads/)
+    - [flask](https://flask.palletsprojects.com/en/2.2.x/installation/#virtual-environments)
```

Comment:
The specific dependencies don't have to be explicitly called out as it's in the requirements.txt file in /ui